sefaria/views.py

```diff
@@ -111,14 +111,14 @@ def accounts(request):
 
 def subscribe(request, email):
     """
-    API for subscribg is mailing lists, in `lists` url param.
+    API for subscribe is mailing lists, in `lists` url param.
```

Comment:
This line is hard to parse, it's possible that `is` needs to change or that the correction should be `subscribing` or...